src/pathfora.js

```diff
@@ -924,8 +979,24 @@
           }
         };
 
+        updateActionCookie = function (name) {
+          var val = utils.readCookie(name),
+              duration = Date.now();
+          var ct;
```

Comment:
not a big deal but u can just `, ct;` here to maintain the var definition pattern